data/hai/unfettered side missions.txt

```diff
@@ -65,3 +65,471 @@ mission "Unfettered returning home"
+			`	Another one steps forward, "Since you so kindly came to Darkcloak to deliver the ship to us, we will allow you to buy back any outfits you may still desire, as a courtesy," they say in a somewhat diplomatic tone.`
+				to display
+					has "flagship planet: Darkcloak"
+			choice
+				`	(Let them have it.)`
+				`	(Take off, now!)`
+					depart
+			`	You and your crew exit what was briefly your ship, and you inform the local Unfettered militia that it is now theirs. The Unfettered that were assembling around your ship happily disperse, except for one, probably the leader judging by their rare dual pulse pistols. They come to talk to you, weapons in hand but lowered. "We are happy you could see reason and decided to hand over the ship. In exchange, we have decided to provide you with a suitable replacement, that should hopefully allow you to exit our space safely.`
+			`	"That relic was not meant for you, but remember that there are other ways for you to prove yourself." The leader of these Unfettered continues with blatantly forced casualness. "Say, for example, bringing us a Jump Drive... or two... that for sure will be earning you favor, and prove you can fight."`
+				to display
+					not "Unfettered Jump Drive 1: offered"
+			branch "has detainers"
+				"outfit: Korath Detainer" > 1
+			action
+				fine 5000000
+			`	As he takes a look at the ship manifest, he seems to notice something missing. "How dare you? You will incur a fine of <fine> for having kept that Korath technology to yourself."`
+				goto nodetain
+			label "has detainers"
+			action
+				outfit "Korath Detainer" -2
+			label nodetain
+			`	"For now, you have enough to worry about with all of the missiles that will be heading your way. If I were you I'd install a lot of jammers. You'll need them, ha!" A general laughter erupts, and they let you get comfortable in the new ship. You may want to get a better one, or at least upgrade this one in order to escape safely. It may be wise to bribes to get out in one piece.`
+			action
+				give ship "Lightning Bug (Unfettered Shipyards)" "Good Will"
+				take ship "Anomalocaris"
+					id "hai legacy"
+
+mission "Unfettered Anomalocaris Lost"
+	landing
+	invisible
+	source
+		attributes "unfettered"
+	to offer
+		has "Hai Legacy 3: done"
+		not "ship model (all): Anomalocaris"
+	on offer
+		conversation
+			`You see the sun slowly disappear into the horizon as you land on this planet, indicating it is already night there. Shortly after exiting your ship, you see a group of Unfettered Hai sitting on the side of the path, eating corn around a campfire. One of them, probably the youngest judging by his size, tries to get your attention by calling you repeatedly. "Captain, Captain! Can you tell me stories about how you used great cunning to beat a swarm of Lightning Bugs?" Another one, probably their parent, then speaks up. "Come, sit with us. I am sure there is much to be said about that great relic from our past."`
```

Comment:
> eating corn around a campfire.

How do the Unfettered have access to a human crop?
```suggestion
			`You see the sun slowly disappear into the horizon as you land on this planet, indicating it is already night there. Shortly after exiting your ship, you see a group of Unfettered Hai sitting on the side of the path, eating corn around a campfire. One of them, probably the youngest judging by his size, sees you and tries to get your attention by calling you repeatedly. "Captain, Captain! Can you tell me stories about how you used great cunning to beat a swarm of Lightning Bugs?" Another one, probably their parent, then speaks up. "Come, sit with us. I am sure there is much to be said about that great relic from our past."`
```

Reply:
the hai have corn as well dont they?
is it named differently or smth?

Reply:
> How do the Unfettered have access to a human crop?

Captured from human merchants during their raids, probably? Either that or acquired from humans by the Hai through trade and then either acquired by the Unfettered in a raid or given to them as 'tribute'.